src/lua/init.c

```diff
@@ -337,7 +338,7 @@ static const char *lua_modules[] = {
  * If there's a risk that a module may fail to load, put it here.
  * Then it'll be embedded, but not loaded until the first use.
  */
-static const char *lua_modules_preload[] = {
+static const char * const lua_modules_preload[] = {
```

Comment:
Creating new `static const char *lua_modules_post_box[]` fails on checkpatch with `ERROR:STATIC_CONST_CHAR_ARRAY: static const char * array should probably be static const char * const`. So I added `static const char *const lua_modules_post_box[]`. After extracting loading to a function, I needed to change `static const char * const lua_modules[]`. Since preload uses the same approach as loaded, its type has been changed for consistency as well.